src/app/event/[id]/page.tsx

```diff
@@ -17,7 +18,14 @@ type EventPageProps = {
 
 const EventPage: NextPage<EventPageProps> = async ({ params }) => {
   const events = (await getAllEvents()) || [];
-  const event = (await getEvent(params?.id)) || defaultEvent;
+  let event = defaultEvent;
+
+  try {
+    event = (await getEvent(params?.id)) || defaultEvent;
```

Comment:
Not sure why we have defaultEvent on event detail page. Why is that needed? If event is not found, I would expect some 404 or something error message

Reply:
This is for typescript. We always need to have type Event here. But It also does not default to event detail page if If event is not found. If will show 404 page if you try to get on non existant event page, př.  `/event/6li0xwokov08`